### PR TITLE
upgrade to Akka.NET v1.4.14 with EventEnvelope updates

### PR DESCRIPTION
**Pull request by Aaronontheweb**

### CODE REVIEW

**Comment by Aaronontheweb** [comment]
@eaba should we merge this PR and have you do a clean-refork before making your other changes?

**Comment by eaba** [comment]
I think "copying" what you have done here will save me some time.....fewer changes to make!

**Comment by Aaronontheweb** [comment]
Sounds good - I'll leave you to it

File: src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs [hunk 1]
```diff
@@ -363,14 +363,23 @@ private Persistent ToPersistenceRepresentation(JournalEntry entry, IActorRef sen
...
-                    sender);
+                    sender,
+                    timestamp: entry.Ordering.Timestamp);
             }
 
             var legacy = entry.SerializerId.HasValue || !string.IsNullOrEmpty(entry.Manifest);
             if (!legacy)
             {
                 var ser = _serialization.FindSerializerForType(typeof(Persistent));
-                return ser.FromBinary<Persistent>((byte[]) entry.Payload);
```
Comment (eaba) [inline src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs:373]: See this.....unreachable code

Comment (Aaronontheweb) [reply src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs:373]: It's not now - I removed it.

Comment (eaba) [reply src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs:373]: Okay

File: src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs [hunk 2]
```diff
@@ -398,12 +407,12 @@ private Persistent ToPersistenceRepresentation(JournalEntry entry, IActorRef sen
                 if (deserialized is Persistent p)
                     return p;
```
Comment (Aaronontheweb) [inline src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs:408]: Need `.WithTimestamp` here

**Pull request closed without merging**